src/Program.cs

```diff
@@ -88,7 +88,20 @@ private T GetXmsCodeGenSetting<T>(CodeModel codeModel, string name)
             foreach (PropertyInfo propertyInfo in typeof(GeneratorSettingsJs).GetProperties())
             {
                 string propertyName = propertyInfo.Name;
-                Settings.Instance.CustomSettings[propertyName] = await GetValue<bool?>(propertyName.ToKebabCase()) ?? false;
+                string kebabCasePropertyName = propertyName.ToKebabCase();

```

Comment:
Did you write ToKebabCase or was it already a thing in AutoRest?

Reply:
I wrote it. It seemed pretty simple.